Sources/BraintreeVenmo/BTVenmoAppSwitchReturnURL.h

```diff
@@ -3,6 +3,7 @@
 extern NSString *const BTVenmoAppSwitchReturnURLErrorDomain;
 
 typedef NS_ENUM(NSUInteger, BTVenmoAppSwitchReturnURLState) {
+    BTVenmoAppSwitchReturnURLStateSucceededWithPaymentContext,
```

Comment:
What is the reasoning for not using `BTVenmoAppSwitchReturnURLStateSucceeded`? This seems to leak details to the merchant that might not be necessary.

Reply:
The ReturnURLState isn't publicly visible to merchants (`BTVenmoAppSwitchReturnURL` is private)